Install.bat

```diff
@@ -0,0 +1,199 @@
+        set "python_path_check=%%i"
+        echo !python_path_check!
+        !python_path_check! --version 2>NUL | findstr /R "3.10" > NUL
+        if %errorlevel% EQU 0 (
+            echo python_path_check | findstr /R "Python310" > NUL
+            if %errorlevel% EQU 0 (
+                set "standalone=!python_path_check!"
+            )
+            echo python_path_check | findstr /R "WindowsApps" > NUL
+            if %errorlevel% EQU 0 (
+                set "WindowsApps=!python_path_check!"
+            )
+        ) else (
+            set "wrong_python=1"
+        )
+    )
+    if NOT "!standalone!"=="" (
+        echo Using standalone Python
+        set "python_path=!standalone!"
+        goto python310
+    ) else if NOT "!WindowsApps!"=="" (
+        echo Using WindowsStore Python
+        set "python_path=!WindowsApps!"
+        goto python310
+    ) 
+    if !wrong_python! EQU 1 (
+        goto wrong_python_display
+    )
+) else (
+    goto wrong_python_display
+)
+:python310
+call:cecho green "Python 3.10.x is installed and running."
+:activate
+if exist "%VENV_DIR%" (  
+    echo Activating virtual environment...
+    call "%VENV_DIR%\Scripts\activate.bat"
+) else (
+    echo Creating virtual environment...
+    !python_path! -m venv %VENV_DIR%
+    goto activate
+)
+:: Upgrading pip  
+python -m pip install --upgrade pip >NUL
+if %errorlevel% equ 0 (
+    call:cecho green "Pip is up-to-date."
+) else (
+   call:cecho green "Pip has been upgraded to the latest version."
+)
+:: Check if requirements.txt exists
+if not exist %requirements_file% (
+    call:cecho red "%requirements_file% does not exist."
+    exit /b 1
+)
+:: Loop through each line in requirements.txt and check if the package is installed
+echo Checking dependencies...
+for /f "tokens=1 delims=~=" %%a in (%requirements_file%) do (
+    pip show %%a >nul 2>&1
+    if errorlevel 1 (
+        call:cecho red "%%a is not installed."
+        ::echo [31m%%a is not installed.[0m
+        set "missing=1"
+    )
+)
+:: Check if any packages are missing
+if defined missing (
+    call:cecho yellow "One or more required packages are missing."
+    :ask_for_choice
+    echo [33m
+    set /p "choice=Do you want to install them? [Y/N]:"
+    echo [0m
+    set "choice=!choice:~0,1!"
+    set "choice=!choice:~0,1!"
+    if /i "!choice!"=="Y" (
+        pip install -r "%requirements_file%"
+        if errorlevel 1 (
+            call:cecho red "Installation failed."
+            goto :ask_for_choice
+        ) else (
+            call:cecho green "Installation successful."
+        )
+    ) else if /i "!choice!"=="N" (
+        echo You chose not to install missing packages.
+        goto again
+    ) else (
+        call:cecho red "Invalid choice. Please enter Y or N."
+        goto :ask_for_choice
+    )
+) else (
+    call:cecho green "All required packages are installed."
+)
+:playwright_check
+pip show playwright >nul 2>&1
+if %errorlevel% equ 0 (
```

Comment:
That doesn't mean that Playwright is fully installed, it needs extra installation: https://reddit-video-maker-bot.netlify.app/docs/installing-playwright